osdep/macosx_application.m

```diff
@@ -68,6 +68,11 @@
         {"cocoa-cb-sw-renderer", OPT_CHOICE(cocoa_cb_sw_renderer,
             {"auto", -1}, {"no", 0}, {"yes", 1})},
         {"cocoa-cb-10bit-context", OPT_FLAG(cocoa_cb_10bit_context)},
+        {"macos-output-csp", OPT_CHOICE(macos_output_csp,
+            {"auto", -1}, {"displayP3_HLG", 0}, {"displayP3_PQ_EOTF", 1}, 
+            {"extendedLinearDisplayP3", 2}, {"itur_2020", 3}, {"itur_2020_HLG", 4}, 
+            {"itur_2020_PQ_EOTF", 5}, {"extendedSRGB",6}, {"extendedLinearSRGB",7}
```

Comment:
the extended color spaces expect component values below 0 and above 1.0. that's what mpv can't do atm and with that those color spaces are a bit useless atm.